addons/OPT/tracker/Events.hpp

```diff
@@ -0,0 +1,37 @@
+//Eventliste
+
+// Event der Ausgelöst wird, wenn ein OPT Karten Dialog geöffnet wird
+#define OPT_KARTEN_DIALOG_OPEN "OPTKartenDialogOpen"
```

Comment:
Würde hier in alle Event-Namen noch den String "EVENT" rein packen.

Also zB `OPT_KARTEN_DIALOG_OPEN_EVENT`

Der OPT-Prefix im Makronamen ist eigentlich auch überflüssig. Dh. man könnte das auch EVENT_KARTEN_DIALOG_OPEN`

Darüber hinaus ist die Mischung aus deutsch und englisch grauselig. Für diesen konkreten Fall schlage ich daher `EVENT_MAP_DIALOG_OPEN`